packages/server/src/modules/reports/typeDefs/dynamic-report.graphql.ts

```
@@ -0,0 +1,46 @@
+
+  " dynamic report data "
+  type DynamicReportInfo {
+    id: ID!
+    name: String!
+    template: [DynamicReportNode!]!
+    created: DateTime!
+    updated: DateTime!
+  }
+
+  " a single node of dynamic report template "
+  type DynamicReportNode {
```

ESLint

[error] type "DynamicReportNode" must have exactly one non-nullable unique identifier.
Accepted name: id.
Accepted types: ID or UUID.

(@graphql-eslint/strict-id-in-types)